lib/src/layout_adjustments/frame.dart

```diff
@@ -9,9 +10,22 @@ class Frame extends StatelessWidget {
     required this.child,
   });
 
+  /// The width of the frame.
+  ///
+  /// If `null`, the intrinsic width of the child is used.
   final double? width;
+
+  /// The height of the frame.
+  ///
+  /// If `null`, the intrinsic height of the child is used.
   final double? height;
+
+  /// The alignment of the child within the frame.
+  ///
+  /// See [Alignment] for more info about the available alignments.
```

Comment:
Can you confirm that `Alignment` includes the same set of values as Swift UI's corresponding alignment that it exposes within `frame()`? For example, I ended up creating custom alignment enumerations for `VStack` and `HStack` because those alignments weren't the same as Flutter's.

Reply:
You're right. `Alignment` is different in Flutter than in SwiftUI. Should we solve this with an extension? Recreate another `Alignment` class? Create another alignment class with a different name?

Should we solve this in the current PR or start a new issue for that?

Reply:
We should solve it in this PR. I suggest creating an enum with the same name and values as Swift UI.